### ISSUE TITLE
Aragog R13.27 > Issues panel > Undefined CAD IDs workflow

### ISSUE THREAD

**Issue by theo-armour** [ladybug-tools/spider#124]
![image](https://user-images.githubusercontent.com/547626/40519401-28f88a5a-5f74-11e8-85a7-0a4bc1f50f84.png)
'
This issue is a placeholder for comments on the workflow for the 'Undefined CAD Object IDs' card in the Issues panel in the left menu.

As of this release the UI is settling down but buttons and inputs are not yet fully functional

The major source of undefined CAD ids appears to be surface type 'Air'. So there is a rule that if no CAD id is found for an Air surface then an is is automatically created.

It is hoped that in future releases there will be more rules like this. The goal is to have a single click fix everything so that the file gets to the analysis app without error.

**Comment by mdengusiak** [comment]
@theo-armour 
I agree but normally these Air surfaces have "overlaps" with shade. And Shade hold information about correct CADObjectID especially when we refer to Revit workflow. So it seems that for this one would be amazing again to look at geometrical data top identify correct CADObjectID. Remember that in many tools tools CADObjectID is place that we assign specific Uvalue, gValues etc... So we need to produce relevant data.
